avr/cores/megacommand/MCL/SeqExtStepPage.cpp

```diff
@@ -534,8 +533,9 @@ void SeqExtStepPage::del_note() {
         uint16_t note_end = j * timing_mid + ev_j.micro_timing - timing_mid;
 
         if ((note_start <= cur_x + cur_w) && (note_end >= cur_x)) {
-          active_track.remove_event(note_idx_on);
+          DEBUG_DUMP("deleteing");
           active_track.remove_event(note_idx_off);
```

Comment:
@yatli  
can you check this.

The note_idx_off event does not get deleted with the reverse ordering. Probably a bug in remove_event.

Reply:
When you delete note_idx_on, the index for off event changes 😅

Reply:
but you have worked out an elegant solution 👍

Reply:
lol an accidental solution.